pkg/controllermanager/controller/project/project_control.go

```diff
@@ -125,6 +127,49 @@ func (r *projectReconciler) reportEvent(project *gardencorev1beta1.Project, isEr
 	r.recorder.Eventf(project, eventType, eventReason, messageFmt, args...)
 }
 
+func (c *Controller) updateLastActivityTimestamp(ctx context.Context, obj interface{}) {
+	objKey, err := cache.MetaNamespaceKeyFunc(obj)
+	if err != nil {
+		logger.Logger.Errorf("Couldn't get key for object %+v: %v", obj, err)
+		return
+	}
+
+	namespace, _, err := cache.SplitMetaNamespaceKey(objKey)
+	if err != nil {
+		logger.Logger.Errorf("Couldn't split namespace key %s due to: %v", objKey, err)
+		return
+	}
+
+	project, err := gutil.ProjectForNamespaceFromReader(ctx, c.gardenClient, namespace)
+	if err != nil {
+		if apierrors.IsNotFound(err) {
+			logger.Logger.Debugf("Secret don`t belong to a project: %v", err)
+			return
+		}
```

Comment:
Do you need this check?
If yes, the error message should not be about secret bot for Project. And it would be nice to include project and namespace names in the error message.